index.html

```diff
@@ -424,5 +424,6 @@ <h4 class="desktop-header">Get started</h4>
       </div>
     </footer>
     <script src="menu.js"></script>
+    <script src="popup.js"></script>
```

Comment:
- [ ] The mobile popup on the mobile view has more space between the buttons and the paragraph, kindly consider fixing this so your design looks more professional and fits all screen sizes properly

![Screenshot_20220908_194202](https://user-images.githubusercontent.com/25227022/189201020-924ed80d-1060-4927-a0c2-60780f57f9b0.png)